### ISSUE TITLE
Need m2 repository

### ISSUE THREAD

**Issue by davsclaus** [mmistretta/RHSummit2018Camel3ScaleLab#1]
I think we need some tutorials on how to setup the maven repository for the Fuse JARs to download.

If you run it vanilla with just Maven you end up with this error:
```
davsclaus:/Users/davsclaus/workspace/RHSummit2018Camel3ScaleLab/summit-example (polish)/$ mvn clean install
[INFO] Scanning for projects...
Downloading from central: https://repo.maven.apache.org/maven2/io/fabric8/fabric8-project-bom-camel-spring-boot/2.3.7.fuse-000036-redhat-2/fabric8-project-bom-camel-spring-boot-2.3.7.fuse-000036-redhat-2.pom
[ERROR] [ERROR] Some problems were encountered while processing the POMs:
[ERROR] Non-resolvable import POM: Could not find artifact io.fabric8:fabric8-project-bom-camel-spring-boot:pom:2.3.7.fuse-000036-redhat-2 in central (https://repo.maven.apache.org/maven2) @ line 29, column 19
[ERROR] 'dependencies.dependency.version' for javax.inject:javax.inject:jar is missing. @ line 40, column 17
[ERROR] 'dependencies.dependency.version' for org.apache.camel:camel-spring-boot-starter:jar is missing. @ line 44, column 17
[ERROR] 'dependencies.dependency.version' for org.apache.camel:camel-swagger-java-starter:jar is missing. @ line 48, column 21
[ERROR] 'dependencies.dependency.version' for org.apache.camel:camel-servlet-starter:jar is missing. @ line 52, column 21
[ERROR] 'dependencies.dependency.version' for org.apache.camel:camel-jackson-starter:jar is missing. @ line 56, column 17
[ERROR] 'dependencies.dependency.version' for org.springframework.boot:spring-boot-starter-web:jar is missing. @ line 60, column 17
[ERROR] 'dependencies.dependency.version' for org.springframework.boot:spring-boot-starter-actuator:jar is missing. @ line 64, column 17
[ERROR] 'dependencies.dependency.version' for junit:junit:jar is missing. @ line 70, column 17
[ERROR] 'dependencies.dependency.version' for org.springframework:spring-test:jar is missing. @ line 75, column 17
[ERROR] 'dependencies.dependency.version' for org.springframework.boot:spring-boot-starter-test:jar is missing. @ line 80, column 17
[ERROR] 'dependencies.dependency.version' for org.jboss.arquillian.junit:arquillian-junit-container:jar is missing. @ line 85, column 17
[ERROR] 'dependencies.dependency.version' for io.fabric8:fabric8-arquillian:jar is missing. @ line 90, column 17
 @
[ERROR] The build could not read 1 project -> [Help 1]
[ERROR]
[ERROR]   The project my.project:summit-example:1.0-SNAPSHOT (/Users/davsclaus/workspace/RHSummit2018Camel3ScaleLab/summit-example/pom.xml) has 13 errors
[ERROR]     Non-resolvable import POM: Could not find artifact io.fabric8:fabric8-project-bom-camel-spring-boot:pom:2.3.7.fuse-000036-redhat-2 in central (https://repo.maven.apache.org/maven2) @ line 29, column 19 -> [Help 2]
[ERROR]     'dependencies.dependency.version' for javax.inject:javax.inject:jar is missing. @ line 40, column 17
[ERROR]     'dependencies.dependency.version' for org.apache.camel:camel-spring-boot-starter:jar is missing. @ line 44, column 17
[ERROR]     'dependencies.dependency.version' for org.apache.camel:camel-swagger-java-starter:jar is missing. @ line 48, column 21
[ERROR]     'dependencies.dependency.version' for org.apache.camel:camel-servlet-starter:jar is missing. @ line 52, column 21
[ERROR]     'dependencies.dependency.version' for org.apache.camel:camel-jackson-starter:jar is missing. @ line 56, column 17
[ERROR]     'dependencies.dependency.version' for org.springframework.boot:spring-boot-starter-web:jar is missing. @ line 60, column 17
[ERROR]     'dependencies.dependency.version' for org.springframework.boot:spring-boot-starter-actuator:jar is missing. @ line 64, column 17
[ERROR]     'dependencies.dependency.version' for junit:junit:jar is missing. @ line 70, column 17
[ERROR]     'dependencies.dependency.version' for org.springframework:spring-test:jar is missing. @ line 75, column 17
[ERROR]     'dependencies.dependency.version' for org.springframework.boot:spring-boot-starter-test:jar is missing. @ line 80, column 17
[ERROR]     'dependencies.dependency.version' for org.jboss.arquillian.junit:arquillian-junit-container:jar is missing. @ line 85, column 17
[ERROR]     'dependencies.dependency.version' for io.fabric8:fabric8-arquillian:jar is missing. @ line 90, column 17
[ERROR]
[ERROR] To see the full stack trace of the errors, re-run Maven with the -e switch.
[ERROR] Re-run Maven using the -X switch to enable full debug logging.
[ERROR]
[ERROR] For more information about the errors and possible solutions, please read the following articles:
[ERROR] [Help 1] http://cwiki.apache.org/confluence/display/MAVEN/ProjectBuildingException
[ERROR] [Help 2] http://cwiki.apache.org/confluence/display/MAVEN/UnresolvableModelException
```

**Comment by davsclaus** [comment]
There is a PR with a fix #4

**Comment by mmistretta** [comment]
resolved via PR
